core.py

```diff
@@ -310,10 +313,49 @@ def list_rule_sets(ctx: click.Context, cache_path: str):
             rule_sets.add(rule_set)
 
 
+@click.command()
+@click.option(
+    "-dp",
+    "--dataset-path",
+    required=True,
+    multiple=True,
+)
+@click.pass_context
+def list_ds_metadata(ctx: click.Context, dataset_path: Tuple[str]):
```

Comment:
can we call this function list_dataset_metadata or just list_metadata? I feel like the ds will confuse users

Reply:
Okay

Reply:
@nhaydel renamed